api/lib/reducers/polkadotV0-reducers.js

```diff
@@ -225,6 +225,10 @@ function getMessageType(section, method) {
   }
 }
 
+function getTransactionMessageAddresses(agentAddress, transactionDetails) {
```

Comment:
if we have to use this function in multiple reducers, we should put it somewhere and import it as opposed to duplicating it.

Reply:
in this case I think it is fine, since all the Polkadot networks would take it from here. But we could also move it to common... although I think it is better here (like `coinReducer` and some others which are exactly the same)

Reply:
across `cosmosV0-source.js` and `polkadotV0-source.js`, I mean

Reply:
I agree with Jordan, please refactor